### DISCUSSION TITLE
How do I create a github pages directory for a custom domain?

### DISCUSSION THREAD

**Discussion by pyl** [community/community#22899] [Other Feature Feedback, Questions, & Ideas]
<p>for example, how do i create <a href="https://qwerty.com/help" rel="nofollow noopener">https://qwerty.com/help</a></p>
<p>when all i have right now is <a href="https://qwerty.com" rel="nofollow noopener">https://qwerty.com</a></p>
<p>I want to avoid using subdomains (<a href="https://help.qwerty.com" rel="nofollow noopener">https://help.qwerty.com</a>)</p>

**Comment by oycyc** [comment, marked as answer]
<p>Hi <a class="mention" href="/python8u">@python8u</a>.</p>
<p>This is quite simple. You can achieve this in two ways.</p>
<p>1.) Simply create a folder inside of the repository that you use for you root domain. Example below.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/b/bb85c93297f1311f2008b94c1f30f1fd0b64b611.jpeg" data-download-href="/uploads/short-url/qKTVabcs5VQXtBGzvlg5VQd43kZ.jpg?dl=1" title="3ADCB8B4-B504-4587-9E1B-A47EF4CA258F.jpeg" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/35899715/181073672-428d90c8-8a26-42f9-85ab-aa2f0aa30a90.jpg" alt="3ADCB8B4-B504-4587-9E1B-A47EF4CA258F.jpeg" data-base62-sha1="qKTVabcs5VQXtBGzvlg5VQd43kZ" width="690" height="388" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/b/bb85c93297f1311f2008b94c1f30f1fd0b64b611_2_690x388.jpg, https://aws1.discourse-cdn.com/github/optimized/2X/b/bb85c93297f1311f2008b94c1f30f1fd0b64b611_2_1035x582.jpg 1.5x, https://aws1.discourse-cdn.com/github/original/2X/b/bb85c93297f1311f2008b94c1f30f1fd0b64b611.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/b/bb85c93297f1311f2008b94c1f30f1fd0b64b611_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">3ADCB8B4-B504-4587-9E1B-A47EF4CA258F.jpeg</span><span class="informations">1136×640 56.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p> As you can see, inside my root domain repository, I have another folder named “contact” and if you go to <a href="https://yangciou.github.io/contact" rel="nofollow noopener">https://yangciou.github.io/contact</a> or <a href="https://yangciou.com/contact" rel="nofollow noopener">https://yangciou.com/contact</a> , it’ll use the code inside that “contact” folder.</p>
<p>2.) Another approach to this is simply by creating a new repository, named whatever you want the path to be(<a href="http://qwerty.com/path" rel="nofollow noopener">qwerty.com/path</a>). After you create a repository with the path’s desired name, you have to go to “Settings” then to find “GitHub Pages” then select the “Source” as what you selected as source in your root domain repository. Then, after pressing “Save,” it should work!</p>
<p>Example: I want <a href="https://qwerty.com/test" rel="nofollow noopener">https://qwerty.com/test</a>, then I would create a repository named “test” and activate GitHub Pages under settings using the instructions above.</p>
